### PR TITLE
Update using-webui.md

### PR DESCRIPTION
**Pull request by OssianMoralis**
Updated the Create Stream via Admin docs page to the latest design.

### CODE REVIEW

File: docs/02-streams-api/getting-started/using-webui.md [hunk 1]
```diff
@@ -3,23 +3,76 @@ title: "Using the Admin Panel"
 slug: "../using-webui"
 description: "Learn how to create a new stream on the Admin Panel of Moralis.io. Follow the step-by-step instructions to monitor your chosen address and receive data through a webhook URL."
 ---
+
+# Start
```
Comment (KoenRijpstra) [inline docs/02-streams-api/getting-started/using-webui.md:7]: Do you want Start to be the title of the page? If not please use `##` to create H2 (second heading)

File: docs/02-streams-api/getting-started/using-webui.md [hunk 2]
```diff
@@ -3,23 +3,76 @@ title: "Using the Admin Panel"
...
 ---
+
+# Start 
+
 1. Go to <http://admin.moralis.io/streams>.
-2. Click on **New Stream** button and following page will open. You can either select some of our predefined template or create your stream from scratch. Let's create from scratch.
+2. Click on **Create a new Stream** button.
+3. Select **Create it from Admin** following page will open. 
+
+<img width="1727" alt="Screenshot 2023-01-11 at 15 37 03" src="https://user-images.githubusercontent.com/122364197/211833828-4fc302a6-5cbd-49b7-bf12-c36000a39385.png">
```
Comment (KoenRijpstra) [inline docs/02-streams-api/getting-started/using-webui.md:13]: Can you store the images in the static\img\content folder as a webp type and the use the `This is a alt text` to place it? 

Let me know if you need help 👍

**Review by KoenRijpstra — CHANGES_REQUESTED** [review]

File: docs/02-streams-api/getting-started/using-webui.md (hunk 1 above)
Comment (OssianMoralis) [reply docs/02-streams-api/getting-started/using-webui.md:7]: No, I think "Create Stream using the Admin Panel" would be the header.

**Review by KoenRijpstra — APPROVED** [review]
LGTM!

**Pull request merged**